Resources/translations/sub_items.en.xliff

```diff
@@ -6,6 +6,41 @@
       <note>The source node in most cases contains the sample message as written by the developer. If it looks like a dot-delimitted string such as "form.label.firstname", then the developer has not provided a default message.</note>
     </header>
     <body>
+      <trans-unit id="35e03083bfdfafc073cf343c74385bec3945b4ac" resname="bulk_delete.error.message">
+        <source>You do not have permission to delete at least 1 of the selected content item(s). Please contact your Administrator to obtain permissions.</source>
+        <target state="new">You do not have permission to delete at least 1 of the selected content item(s). Please contact your Administrator to obtain permissions.</target>
+        <note>key: bulk_delete.error.message</note>
+      </trans-unit>
+      <trans-unit id="410f2ebb276d839b6606da042d0327a32e949093" resname="bulk_delete.popup.cancel">
+        <source>Cancel</source>
+        <target state="new">Cancel</target>
+        <note>key: bulk_delete.popup.cancel</note>
+      </trans-unit>
+      <trans-unit id="a668a561527cc0e3e2c7952fbf300558d9d4f877" resname="bulk_delete.popup.confirm">
+        <source>Send to trash</source>
+        <target state="new">Send to trash</target>
+        <note>key: bulk_delete.popup.confirm</note>
+      </trans-unit>
+      <trans-unit id="02d93b03435046c6bcf08ce15ae7970f7ff8f746" resname="bulk_delete.popup.message">
+        <source>Are you sure you want to send the selected content item(s) to the trash?</source>
```

Comment:
`<source>Are you sure you want to send the selected content item(s) to trash?</source>`
`to trash`, for consistency with other strings.

Reply:
Good pick, @DominikaK, indeed even in inVision prototypes there is `to trash`. My mistake. :-)